GooglePay/src/main/java/com/braintreepayments/api/GooglePayRequest.java

```diff
@@ -172,13 +175,17 @@ public void setAllowedCardNetworks(@NonNull String paymentMethodType, @NonNull J
     }
 
     /**
+     * Optional.
```

Comment:
can we add `@Deprecated` on the method and `@deprecated` on the jdoc here just to enforce they'll be going away in the next major version?

Reply:
Good call - added the deprecation tags and an CHANGELOG entry to make it explicit